tests/test_traits.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+import logging 
+from traitlets import HasTraits, TraitError
+from jupyter_telemetry.traits import HandlersList
+
+
+class HasHandlersList(HasTraits):
+    handlers_list = HandlersList(
+        None,
+        allow_none=True
+    )
+
+
+def test_good_handlers_list_value():
+    handlers = [
+        logging.NullHandler(), 
+        logging.NullHandler()
+    ]
+    obj = HasHandlersList(
+        handlers_list=handlers
+    )
+    assert obj.handlers_list
```

Comment:
Can we also assert that calling them returns the same list?

Reply:
Ah, yes. 🤦‍♂ I forgot to finish this line.